### ISSUE TITLE
MO 2.0.4a sends mod to bottom when renaming

### ISSUE THREAD

**Issue by striker873** [TanninOne/modorganizer#581]
Another issue I found: when renamin a Mod (left pane) MO changes both name and priority, giving the mod a new number, and consecuently sending it to the very bottom of the priority order.

**Comment by LawLeewer** [comment]
It not only sends it to the bottom of the priority order, but it also deactivates the mod. 
And even though the mod is deactivated the esps still appear ticked on the load order.

**Comment by striker873** [comment]
Yep LawLeewer, it does exactly what you say, I checked again. When renaming a mod, MO changes priority to lowest and also uncheckes mod (left panel) but esp is still checked on loadorder.
